### PR TITLE
UI : Localization fix

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

-  Localization fix

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 1]
```diff
@@ -121,6 +121,7 @@
     "configure-dbt-model": "Configure dbt Model",
     "configure-entity": "Configure {{entity}}",
     "configure-glossary-term": "Configure Glossary Term",
+    "configure-test-case": "Configure Test Case",
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:124]:
```suggestion
    "configure-test-case": "Configure test case",
```

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 2]
```diff
@@ -291,6 +296,8 @@
     "field-required": "{{field}} is required",
     "field-required-plural": "{{field}} are required",
     "filter-plural": "Filters",
+    "first-capitalize": "First",
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:299]:
```suggestion
    "first": "First",
```

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 3]
```diff
@@ -477,8 +486,10 @@
     "partitions": "Partitions",
     "passed": "Passed",
     "password": "Password",
+    "password-lowercase": "password",
     "password-not-match": "Password Doesn't Match",
     "password-type": "{{type}} Password",
+    "password-type-first": "Please type password first",
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:492]:
```suggestion
    "please-password-type-first": "Please type password first",
```

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by karanh37 — DISMISSED** [review]
We should be covering the tests for useTranslation with actual values.
Have raised with @Sachin-chaurasiya , he will be covering in the other localization PR.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**